ang/test/civicase/case/details/directives/case-details.directive.spec.js

```diff
@@ -48,14 +53,45 @@
     });
 
     describe('focusToggle()', function () {
-      beforeEach(function () {
-        compileDirective();
-        element.isolateScope().isFocused = true;
-        element.isolateScope().focusToggle();
+      describe('basic test', function () {
+        beforeEach(function () {
+          compileDirective();
+          element.isolateScope().isFocused = true;
+          element.isolateScope().focusToggle();
+        });
+
+        it('toggles the focus state', function () {
+          expect(element.isolateScope().isFocused).toBe(false);
+        });
       });
 
-      it('toggles the focus state', function () {
-        expect(element.isolateScope().isFocused).toBe(false);
+      describe('when case is unfocused and screen width is less than 1690px', function () {
+        beforeEach(function () {
+          spyOn($rootScope, '$broadcast');
+          spyOn($document, 'width').and.returnValue(1600);
```

Comment:
Please make sure to revert the `$document.width()` method back to the original. This can cause problems for other tests.

Reply:
This is not necessary, because I am spying on `$document` and not `document`. The value gets reset automatically when the test is over, I have tested it.

Reply:
oh right!!!!!!!!